/**/riptide_mapping2:
  ros__parameters:
    init_data:
      gate:
        parent: map
        covar:
          x: 1.0
          y: 1.0
          z: 1.0
          yaw: 1.0
        pose:
          x: 3.5
          y: 0.0
          z: -0.35
          yaw: 180.0
      earth_glyph:
        parent: gate_frame
        covar:
          x: 1.0
          y: 1.0
          z: 1.0
          yaw: 1.0
        pose:
          x: 0.0
          y: 0.25
          z: 0.0
          yaw: 0.0
      buoy:
        parent: map
        covar:
          x: 1.0
          y: 1.0
          z: 1.0
          yaw: 1.0
        pose:
          x: 5.0
          y: -3.0
          z: -1.0
          yaw: 180.0
      buoy_glyph_1:
        parent: buoy_frame
        covar:
          x: 1.0
          y: 1.0
          z: 1.0
          yaw: 1.0
        pose:
          x: 0.0
          y: 0.0
          z: 0.0
          yaw: 0.0
      buoy_glyph_2:
        parent: buoy_frame
        covar:
          x: 1.0
          y: 1.0
          z: 1.0
          yaw: 1.0
        pose:
          x: 0.0
          y: 0.0
          z: 0.0
          yaw: 0.0
      torpedo:
        parent: map
        covar:
          x: 1.0
          y: 1.0
          z: 1.0
          yaw: 1.0
        pose:
          x: 4.0
          y: 3.0
          z: -1.0
          yaw: 180.0
      torpedo_upper_hole:
        parent: torpedo_frame
        covar:
          x: 1.0
          y: 1.0
          z: 1.0
          yaw: 1.0
        pose:
          x: 0.0
          y: 0.0
          z: 0.25
          yaw: 0.0
      torpedo_lower_hole:
        parent: torpedo_frame
        covar:
          x: 1.0
          y: 1.0
          z: 1.0
          yaw: 1.0
        pose:
          x: 0.0
          y: 0.0
          z: -0.25
          yaw: 0.0

    # Covariance filter parameters
    cov_limit: 0.01 # Minimum value for the covariance matrix elements
    k_value: 0.01   # Covariance step size (< 1.0) applied per detection
    detection_cov_factor: 7.0

    # relative position filter parameters
    angle_cutoff: 15.0
    distance_limit: 10.0
    confidence_cutoff: 0.7
        
  
